reference/append.rst

```diff
@@ -207,8 +207,8 @@ Scaling Data Ingestion
 
 If your use-case does not require real-time ingests, then using append distributed tables will give you the highest ingest rates. This approach is more suitable for use-cases which use time-series data and where the database can be a few minutes or more behind.
 
-Master Node Bulk Ingestion (100k/s-200k/s)
-$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+Coordinator Node Bulk Ingestion (100k/s-200k/s)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
 To ingest data into an append distributed table, you can use the `COPY <http://www.postgresql.org/docs/current/static/sql-copy.html>`_ command, which will create a new shard out of the data you ingest. COPY can break up files larger than the configured citus.shard_max_size into multiple shards. COPY for append distributed tables only opens connections for the new shards, which means it behaves a bit differently than COPY for hash distributed tables, which may open connections for all shards. A COPY for append distributed tables command does not ingest rows in parallel over many connections, but it is safe to run many commands in parallel.
 
```

Comment:
Below, could we update:
``` SQL

     -- In a separate transaction, append the staging table
     SELECT master_append_table_to_shard(select_events_shard(), 'stage_1', 'master-node', 5432);
```

and update the 'master-node' value with something like 'coordinator-node-101'?